tests/test_panorama/test_panorama_device.py

```diff
@@ -0,0 +1,423 @@
+            Name="test-device-name",
+            NetworkingConfiguration={
+                "Ethernet0": {
+                    "ConnectionType": "STATIC_IP",
+                    "StaticIpConnectionInfo": {
+                        "DefaultGateway": "192.168.1.1",
+                        "Dns": [
+                            "8.8.8.8",
+                        ],
+                        "IpAddress": "192.168.1.10",
+                        "Mask": "255.255.255.0",
+                    },
+                },
+                "Ethernet1": {
+                    "ConnectionType": "dhcp",
+                },
+                "Ntp": {
+                    "NtpServers": [
+                        "0.pool.ntp.org",
+                        "1.pool.ntp.org",
+                        "0.fr.pool.ntp.org",
+                    ]
+                },
+            },
+            Tags={"Key": "test-key", "Value": "test-value"},
+        )
+
+    resp = client.describe_device(DeviceId=resp["DeviceId"])
+
+    assert resp["AlternateSoftwares"] == [{"Version": "0.2.1"}]
+    assert (
+        resp["Arn"] == "arn:aws:panorama:eu-west-1:123456789012:device/test-device-name"
+    )
+    assert resp["Brand"] == "AWS_PANORAMA"
+    assert resp["CreatedTime"] == datetime(2020, 1, 1, 12, 0, tzinfo=tzutc())
+    assert resp["CurrentNetworkingStatus"] == {
+        "Ethernet0Status": {
+            "ConnectionStatus": "CONNECTED",
+            "HwAddress": "8C:0F:5F:60:F5:C4",
+            "IpAddress": "192.168.1.300/24",
+        },
+        "Ethernet1Status": {
+            "ConnectionStatus": "NOT_CONNECTED",
+            "HwAddress": "8C:0F:6F:60:F4:F1",
+            "IpAddress": "--",
+        },
+        "LastUpdatedTime": datetime(2020, 1, 1, 12, 0, tzinfo=tzutc()),
+        "NtpStatus": {
+            "ConnectionStatus": "CONNECTED",
+            "IpAddress": "91.224.149.41:123",
+            "NtpServerName": "0.pool.ntp.org",
+        },
+    }
+    assert resp["CurrentSoftware"] == "6.2.1"
+    assert resp["Description"] == "test device description"
+    assert resp["DeviceAggregatedStatus"] == "ONLINE"
+    assert resp["DeviceConnectionStatus"] == "ONLINE"
+    assert resp["DeviceId"] == "device-RsozEWjZpeNe3SXHidX3mg=="
+    assert resp["LatestDeviceJob"] == {"JobType": "REBOOT", "Status": "COMPLETED"}
+    assert resp["LatestSoftware"] == "6.2.1"
+    assert resp["LeaseExpirationTime"] == datetime(2020, 1, 6, 12, 0, tzinfo=tzutc())
+    assert resp["Name"] == "test-device-name"
+    assert resp["ProvisioningStatus"] == "SUCCEEDED"
+    assert resp["SerialNumber"] == "GAD81E29013274749"
+    assert resp["Tags"] == {"Key": "test-key", "Value": "test-value"}
+    assert resp["Type"] == "PANORAMA_APPLIANCE"
+
+
+@mock_panorama
+def test_provision_device_aggregated_status_lifecycle() -> None:
+    client = boto3.client("panorama", region_name="eu-west-1")
+    given_device_name = "test-device-name"
+    state_manager.set_transition(
```

Comment:
This test would have to be skipped in ServerMode as well, as the `state_manager`-object only works in decorator-mode.

(If you want to configure this in ServerMode, you would have to make a HTTP-request to `http://localhost:5000/moto-api/state-manager/set-transition` instead - but I don't think that adds anything here.)

Reply:
Done: https://github.com/getmoto/moto/pull/6948/commits/1c0dd6474c8970c06c9d6e24048495029a91a366